debugview/plugin.cpp

```diff
@@ -102,37 +103,76 @@ class debugview : public threadloop {
 				}
 			}
 			ImGui::SameLine();
 			ImGui::Text("Resets to zero'd out tracking universe");
 		}
 		ImGui::Spacing();
 		ImGui::Text("Switchboard connection status:");
+		ImGui::Text("Predicted pose topic:");
+		ImGui::SameLine();
+
+		if (pp->fast_pose_reliable()) {
+			const pose_type predicted_pose = pp->get_fast_pose().pose;
+			ImGui::TextColored(ImVec4(0.0, 1.0, 0.0, 1.0), "Valid predicted pose pointer");
+			ImGui::Text("Prediced pose position (XYZ):\n  (%f, %f, %f)", predicted_pose.position.x(), predicted_pose.position.y(), predicted_pose.position.z());
+			ImGui::Text("Predicted pose quaternion (XYZW):\n  (%f, %f, %f, %f)", predicted_pose.orientation.x(), predicted_pose.orientation.y(), predicted_pose.orientation.z(), predicted_pose.orientation.w());
+		} else {
+			ImGui::TextColored(ImVec4(1.0, 0.0, 0.0, 1.0), "Invalid predicted pose pointer");
+		}
+
 		ImGui::Text("Fast pose topic:");
 		ImGui::SameLine();
 
-		if(pp->fast_pose_reliable()) {
-			const pose_type fast_pose = pp->get_fast_pose().pose;
+		const imu_raw_type *raw_imu = _m_fast_pose->get_latest_ro();
+		if (raw_imu) {
+			pose_type raw_pose;
+			raw_pose.position = Eigen::Vector3f{float(raw_imu->pos(0)), float(raw_imu->pos(1)), float(raw_imu->pos(2))};
+            raw_pose.orientation = Eigen::Quaternionf{float(raw_imu->quat.w()), float(raw_imu->quat.x()), float(raw_imu->quat.y()), float(raw_imu->quat.z())};
+		
+			pose_type swapped_pose;
+			swapped_pose.position.x() = -raw_pose.position.y();
+			swapped_pose.position.y() = raw_pose.position.z();
+			swapped_pose.position.z() = -raw_pose.position.x();
+
+			Eigen::Quaternionf raw_o (raw_pose.orientation.w(), -raw_pose.orientation.y(), raw_pose.orientation.z(), -raw_pose.orientation.x());
+			swapped_pose.orientation = raw_o * pp->get_offset();
+
 			ImGui::TextColored(ImVec4(0.0, 1.0, 0.0, 1.0), "Valid fast pose pointer");
-			ImGui::Text("Fast pose position (XYZ):\n  (%f, %f, %f)", fast_pose.position.x(), fast_pose.position.y(), fast_pose.position.z());
-			ImGui::Text("Fast pose quaternion (XYZW):\n  (%f, %f, %f, %f)", fast_pose.orientation.x(), fast_pose.orientation.y(), fast_pose.orientation.z(), fast_pose.orientation.w());
+			ImGui::Text("Fast pose position (XYZ):\n  (%f, %f, %f)", swapped_pose.position.x(), swapped_pose.position.y(), swapped_pose.position.z());
+			ImGui::Text("Fast pose quaternion (XYZW):\n  (%f, %f, %f, %f)", swapped_pose.orientation.x(), swapped_pose.orientation.y(), swapped_pose.orientation.z(), swapped_pose.orientation.w());
 		} else {
 			ImGui::TextColored(ImVec4(1.0, 0.0, 0.0, 1.0), "Invalid fast pose pointer");
 		}
 
 		ImGui::Text("Slow pose topic:");
 		ImGui::SameLine();
 
-		const pose_type* slow_pose_ptr = _m_slow_pose->get_latest_ro();
-		if(slow_pose_ptr){
+		const pose_type *slow_pose_ptr = _m_slow_pose->get_latest_ro();
+		if (slow_pose_ptr){
+			pose_type swapped_pose;
+
+			// This uses the OpenVINS standard output coordinate system.
```

Comment:
Instead of calling it the OpenVINS coordinate system can we actually just describe the coordinate system. This will be useful when porting other SLAMs that might have other coordinate systems.